src/main/resources/assets/admin/common/js/form/FormContext.ts

```diff
@@ -11,10 +12,13 @@ export class FormContext {
 
     private language: string;
 
+    private customValidationErrors: ValidationError[];
```

Comment:
Let's rename to `validationErrors`. It's not only custom (originated from an app) errors that will be here, system errors might be here as well.